### PR TITLE
Updated readme file with better design, less text

### PR DESCRIPTION
**Pull request by ServiAmirPM**
Still requires more modeifications (results, features, tutorials)

## Pull Request Description:

- Re-structure - changed the way the document is structured (table of content), including changing titles names
- Redesigned the header - with better look and feel, with as much information in the header.
- Removed tutorials from the tutorials section in the readme - will add some back after re-structuring the tutorials.
- Added visuals to the page for better look and feel, and to better explain the product.
- improved the quantization flow table and added 4 tutorials to it.
- Re-ordered the Resources section, making it clear that API is per quantization method
- Hidden (collapsed) the supported version table. users can open it on demand
- Supported features - didn't touch it - still needs work
- Quantization - didn't touch it - still needs work
- Results - removed text, still requires updating the numbers, add mixed percision and the update graph
- Toubleshooting and community - added community to this section.
- Updated the license section with text and a badge

## Checklist before requesting a review:
- [ ] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

**Comment by ofirgo** [comment]
Please provide more details regarding the changes in the PR description (what are the main changes to the readme, what was added/removed, etc.), both for documentation and to make my review easier 😄

**Comment by ServiAmirPM** [comment]
- Re-structure - changed the way the document is structured (table of content), including changing titles names 
- Redesigned the header - with better look and feel, with as much information in the header.
- Removed tutorials from the tutorials section in the readme - will add some back after re-structuring the tutorials.
- Added visuals to the page for better look and feel, and to better explain the product.
- improved the quantization flow table and added 4 tutorials to it.
- Re-ordered the Resources section, making it clear that API is per quantization method
-  Hidden (collapsed) the supported version table. users can open it on demand
- Supported features - didn't touch it - still needs work
- Quantization - didn't touch it - still needs work
- Results - removed text, still requires updating the numbers, add mixed percision and the update graph
- Toubleshooting and community - added community to this section. 
- Updated the license section with text and a badge

File: README.md [hunk 1]
```diff
@@ -1,55 +1,104 @@
...
+* [Debug](https://sony.github.io/model_optimization/docs/guidelines/visualization.html) – modify optimization process or generate explainable report
+  
+* [Release notes](https://github.com/sony/model_optimization/releases)
 
 
 ### Supported Versions
 
 Currently, MCT is being tested on various Python, Pytorch and TensorFlow versions:
-
+<details id="supported-versions">
```
Comment (ofirgo) [inline README.md:86]: I believe you have a conflict here with the current readme - we added Python 3.12 and Pytorch 2.5 to this table.
Maybe remove torch 2.1 so it won't be too much (also remove python 3.9 from the table?)
Let me know if you need help merging and solving the conflicts

File: README.md [hunk 2]
```diff
@@ -83,7 +133,7 @@ MCT supports different quantization methods:
 | Quantization Method                           | Complexity | Computational Cost          |
 |-----------------------------------------------|------------|-----------------------------|
 | PTQ                                           | Low        | Low (order of minutes)      |
-| GPTQ (parameters fine-tuning using gradients) | Mild       | Mild (order of 2-3 hours)   |
+| GPTQ (parameters fine-tuning using gradients) | Moderate   | Moderate (order of 2-3 hours)   |
 | QAT                                           | High       | High (order of 12-36 hours) |
```
Comment (ofirgo) [inline README.md:123]: the "Main features:" part below is unreadable, let's think how we can improve it

File: README.md [hunk 3]
```diff
@@ -83,7 +133,7 @@ MCT supports different quantization methods:
...
```
Comment (ofirgo) [inline README.md:124]: Also, need to fix the reference and title for the EPTQ paper

File: README.md [hunk 4]
```diff
@@ -148,6 +191,14 @@ In the following table we present the ImageNet validation results for these mode
 | ResNet-18 [3]             | 69.86           | 69.63           |69.53|
 | SqueezeNet 1.1 [3]        | 58.128          | 57.678          ||
 
+### Keras
+MCT can quantize an existing 32-bit floating-point model to an 8-bit fixed-point (or less) model without compromising accuracy. 
+Below is a graph of [MobileNetV2](https://keras.io/api/applications/mobilenet/) accuracy on ImageNet vs average bit-width of weights (X-axis), using 
+single-precision quantization, mixed-precision quantization, and mixed-precision quantization with GPTQ. 
+
+<img src="https://github.com/sony/model_optimization/raw/main/docsrc/images/mbv2_accuracy_graph.png">
```
Comment (ofirgo) [inline README.md:191]: we are preparing this results figure in Pytorch, so need to think what are we going to include under the keras section results

File: README.md [hunk 5]
```diff
@@ -148,6 +191,14 @@ In the following table we present the ImageNet validation results for these mode
...
+
+For more results, please see [1]
```
Comment (ofirgo) [inline README.md:193]: This is a very old not very relevant reference (for results purposes).
If we need more results maybe we can try to create a separate table and reference to it, but I'm not sure that the results in this paper are representing for the current performance of MCT

File: README.md [hunk 6]
```diff
@@ -158,23 +209,25 @@ Results for applying pruning to reduce the parameters of the following models by
 | ResNet50 [2]    | 75.1                 | 72.4                  |
 | DenseNet121 [3] | 74.44                | 71.71                 |
 
-
-## Trouble Shooting
+## Troubleshooting and Community
 
 If the accuracy degradation of the quantized model is too large for your application, check out the [Quantization Troubleshooting](https://github.com/sony/model_optimization/tree/main/quantization_troubleshooting.md)
```
Comment (ofirgo) [inline README.md:206]: "If the accuracy degradation of the quantized model is too large for your application" -->
"If you encountered large accuracy degradation from the quantized model returned by MCT..."

File: README.md [hunk 7]
```diff
@@ -158,23 +209,25 @@ Results for applying pruning to reduce the parameters of the following models by
...
+Check out the [FAQ](https://github.com/sony/model_optimization/tree/main/FAQ.md) for common issues. 
+
+You are welcome to ask questions and get support on our [issues section](https://github.com/sony/model_optimization/issues) and manage community discussions under [discussions section](https://github.com/sony/model_optimization/discussions).
 
 
 ## Contributions
 MCT aims at keeping a more up-to-date fork and welcomes contributions from anyone.
 
-*You will find more information about contributions in the [Contribution guide](https://github.com/sony/model_optimization/blob/main/CONTRIBUTING.md).
+*Checkout more our [Contribution guide](https://github.com/sony/model_optimization/blob/main/CONTRIBUTING.md) for more details.
```
Comment (ofirgo) [inline README.md:217]: remove "more"

**Review by ofirgo — CHANGES_REQUESTED** [review]

File: README.md [hunk 8]
```diff
@@ -1,55 +1,104 @@
...
+<p align="center">
+  <a href="#getting-started">Getting Started</a> •
+  <a href="#tutorials-and-examples">Tutorials</a> •
+  <a href="#supported-features">High level features and techniques</a> •
+  <a href="#resources">Resources</a> • 
+  <a href="#contributions">Community</a> •
+  <a href="#license">License</a>
+</p>
+<p align="center">
+  <a href="https://sony.github.io/model_optimization#prerequisites"><img src="https://img.shields.io/badge/python-2.1%20%7C%202.2%20%7C%202.3-blue" /></a>
```
[the remark this answers is not in the document]
Comment (ServiAmirPM) [reply README.md]: Solved already

File: README.md [hunk 9]
```diff
@@ -1,55 +1,104 @@
...
-## Table of Contents
+### Tutorials and Examples 
 
-- [Getting Started](https://github.com/sony/model_optimization?tab=readme-ov-file#getting-started)
-- [Supported features](https://github.com/sony/model_optimization?tab=readme-ov-file#supported-features)
-- [Results](https://github.com/sony/model_optimization?tab=readme-ov-file#results)
-- [Troubleshooting](https://github.com/sony/model_optimization?tab=readme-ov-file#trouble-shooting)
-- [Contributions](https://github.com/sony/model_optimization?tab=readme-ov-file#contributions)
-- [License](https://github.com/sony/model_optimization?tab=readme-ov-file#license)
+Our [tutorials](https://github.com/sony/model_optimization/blob/main/tutorials/README.md) section will walk you through the basics of deploying your first model; Covering various compression techniques for both Keras and PyTorch models.
```
[the remark this answers is not in the document]
Comment (ServiAmirPM) [reply README.md]: Done. Committed fix.

File: README.md [hunk 10]
```diff
@@ -1,55 +1,104 @@
...
+Our [tutorials](https://github.com/sony/model_optimization/blob/main/tutorials/README.md) section will walk you through the basics of deploying your first model; Covering various compression techniques for both Keras and PyTorch models. 
+Access interactive notebooks for hands-on learning with popular models/tasks or move on to [Resources](#resources) section.
 
+### Supported Quantization flows</div>  
+MCT supports various quantization flows as appears below. 
+<div align="center">
+<p align="center">
 
-## Getting Started
+  Quantization Method  | Complexity | Computational Cost | Tutorial
```
[the remark this answers is not in the document]
Comment (ServiAmirPM) [reply README.md:50]: Resolved. removed the grey ones.

File: README.md [hunk 11]
```diff
@@ -1,55 +1,104 @@
...
+MCT supports various quantization flows as appears below. 
+<div align="center">
+<p align="center">
 
-## Getting Started
+  Quantization Method  | Complexity | Computational Cost | Tutorial 
+-------------------- | -----------|--------------------|---------
+PTQ (Post Training Quantization)  | Low | Low (~1-10 CPU minutes) | <a href="https://colab.research.google.com/github/sony/model_optimization/blob/main/tutorials/notebooks/mct_features_notebooks/pytorch/example_pytorch_post_training_quantization.ipynb"><img src="https://img.shields.io/badge/Pytorch-green"/></a> <a href="https://colab.research.google.com/github/sony/model_optimization/blob/main/tutorials/notebooks/mct_features_notebooks/keras/example_keras_post-training_quantization.ipynb"><img src="https://img.shields.io/badge/Keras-green"/></a>
+GPTQ (parameters fine-tuning using gradients)  | Moderate | Moderate (~2-3 GPU hours) | <a href="https://colab.research.google.com/github/sony/model_optimization/blob/main/tutorials/notebooks/mct_features_notebooks/keras/example_keras_mobilenet_gptq.ipynb"><img src="https://img.shields.io/badge/Keras-green"/></a> <img src="https://img.shields.io/badge/Pytorch-grey"/>
```
[the remark this answers is not in the document]
Comment (ServiAmirPM) [reply README.md]: fixed

File: README.md [hunk 12]
```diff
@@ -1,55 +1,104 @@
...
+QAT (Quantization Aware Training)  | High | High (~12-36 GPU hours) | <a href="https://colab.research.google.com/github/sony/model_optimization/blob/main/tutorials/notebooks/mct_features_notebooks/keras/example_keras_qat.ipynb"><img src="https://img.shields.io/badge/Keras-green"/></a> <img src="https://img.shields.io/badge/Pytorch-grey"/>
 
-This section provides an installation and a quick starting guide.
+</p>    
+</div>
 
-### Installation
+For each flow, **Quantization core** utilizes various algorithms and hyper-parameters for optimal [hardware-aware](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/target_platform_capabilities/README.md) quantization results. 
+For further details, please see [Supported features and algorithms](#supported-features). User can either provide their own representative dataset, or utilize the [Data Generation](#data-generation-) capability.
```
[the remark this answers is not in the document]
Comment (ServiAmirPM) [reply README.md:60]: Resolved

File: README.md [hunk 13]
```diff
@@ -1,55 +1,104 @@
...
 
-To install the latest stable release of MCT, run the following command:
-```
-pip install model-compression-toolkit
-```
+<div align="center">
+<p align="center">
 
-For installing the nightly version or installing from source, refer to the [installation guide](https://github.com/sony/model_optimization/blob/main/INSTALLATION.md).
+<img src="/docsrc/images/mctFlow.png" width="800">
```
[the remark this answers is not in the document]
Comment (ServiAmirPM) [reply README.md:65]: Will upgrade in upcoming future (designer will provide better one)

File: README.md [hunk 14]
```diff
@@ -1,55 +1,104 @@
...
+</p>    
+</div>
 
+### Resources
+* [User Guide](https://sony.github.io/model_optimization/docs/index.html)  contains detailed information about MCT and guides you from installation through optimizing models for your edge AI applications.
 
-### Quick start & tutorials 
+* MCT's [API Docs](https://sony.github.io/model_optimization/docs/api/api_docs/) is seperated per quantization methods:
```
[the remark this answers is not in the document]
Comment (ServiAmirPM) [reply README.md:72]: fixed

File: README.md (hunk 2 above)
Comment (ServiAmirPM) [reply README.md:123]: Agree - will fix in next round.

File: README.md (hunk 4 above)
Comment (ServiAmirPM) [reply README.md:191]: Agree - next round.

File: README.md (hunk 6 above)
Comment (ServiAmirPM) [reply README.md:206]: fixed

File: README.md (hunk 7 above)
Comment (ServiAmirPM) [reply README.md:217]: fixed
